cloudsim-plus-testbeds/src/main/java/org/cloudsimplus/hostfaultinjection/HostFaultInjectionExperiment.java

```diff
@@ -224,11 +281,11 @@ private double getMaxNumberOfVmsForCustomerExpectedPrice(DatacenterBroker broker
 
     @Override
     protected List<Vm> createVms(DatacenterBroker broker) {
-        final int numVms = (int) getTemplatesMap().get(broker).getMaxNumberOfVmsForCustomer();
+        numVms = getContract(broker).getMinFaultToleranceLevel();
```

Comment:
It is as an attribute of the class, since from the value of the amount of vms that have been set the cloudlets will be defined.